incubator/hnc/pkg/kubectl/tree.go

```diff
@@ -34,15 +36,24 @@ var (
 var treeCmd = &cobra.Command{
 	Use:   "tree",
 	Short: "Displays the hierarchy tree rooted at the given namespace",
-	Args:  cobra.ExactArgs(1),
+	Args:  cobra.MaximumNArgs(1),
 	Run: func(cmd *cobra.Command, args []string) {
-		nnm := args[0]
 		footnotesByMsg = map[string]int{}
 		footnotes = []string{}
-		hier := getHierarchy(nnm)
-		fmt.Println(nameAndFootnotes(hier))
-		printSubtree("", hier)
-
+		nsList := make([]string, 0)
+		if len(args) != 0 {
```

Comment:
Now I'm wondering if we should allow _multiple_ namespaces to be provided by the user? Eg, I could say `kubectl hnc tree foo bar baz` and it would show me the trees rooted at each of these namespaces. There could be some duplication but I think that's fine. Would you please make this change (and update the help text)?